### DISCUSSION TITLE
How to use TLS authentication with entity operator?

### DISCUSSION THREAD

**Discussion by ksa75** [strimzi/strimzi-kafka-operator#9477] [Q&A]
Since I enable authentication on ingress listener my new topics/users never get ready with warning in log:

kc -n kafka logs cluster0-entity-operator-6594568847-l6dv7
..
WARN [vertx-blocked-thread-checker] BlockedThreadChecker: - Thread Thread[blocking-startup-ops-0,5,main] has been blocked for 60122 ms, time limit is 60000 ms
io.vertx.core.VertxException: Thread blocked

So I have to rollback to get it working
My cluster (breaking changes are commented):

        1 apiVersion: kafka.strimzi.io/v1beta2
       2 kind: Kafka
       3 metadata:
       4   name: cluster0
       5 spec:
       6   kafka:
       7     version: 3.2.1
       8     replicas: 3
       9     authorization:
      10       type: simple
      11       superUsers:
      12       - CN=superuser
      13     listeners:
      14       - name: tls
      15         port: 9093
      16         type: internal
      17         tls: true
      18         authentication:
      19           type: tls
      20       - name: external
      21         port: 9094
      22         type: ingress
      23         # authentication:
      24           # type: tls
      25         tls: true
      26         configuration:
      27           bootstrap:
    ..

What Ive missing? Should I prepare entity operator somehow for enabling authentication on ingress listener

**Comment by scholzj** [comment]
The Entity Operator does not use any of the listeners you configure and does not care about what kind of authentication you use there.

**Comment by ksa75** [comment]
Thanks. It had nothing with ingress. Just a matter of time. topics/users got ready  eventually
